src/sonic-config-engine/sonic-cfggen

```diff
@@ -317,9 +317,6 @@ def main():
         if args.port_config is None:
             args.port_config = device_info.get_path_to_port_config_file(hwsku)
         (ports, _, _) = get_port_config(hwsku, platform, args.port_config, asic_id)
-        if not ports:
```

Comment:
>ports [](http://example.com/codeflow?start=15&length=5)

This is a new feature or significant fix. Could you add new testcase to protect it?

Reply:
Still applicable. Could you help add a positive testcase?

Reply:
I am on it.

Reply:
@qiluo-msft
While I implemented the unittest I noticed a bug.
I handled the flow of reading the ports from port_config.ini but not the flow of reading the ports from platrfom.json+hwsku.json. Now it is fixed and I added unittest as well.
Appreciate your review.